app/services/weet.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+const API_GEEKS = 'https://geek-jokes.sameerkumar.website/api';
```

Comment:
This variable must go in the environment variables, then you add it in `config/index.js` and then require it from here.